### DISCUSSION TITLE
Buttplug.io support

### DISCUSSION THREAD

**Discussion by scottay1127** [stashapp/stash#2137] [Ideas]
I see that the Handy has been integrated - any plans to integrate Buttplug.io? That would open up the ability for many, many more interactive stroker devices.

**Comment by WithoutPants** [comment]
The Handy integration was (presumably) submitted by someone with an applicable appliance. I don't personally have any equipment to test with (and as a result no personal plan to add the functionality). It will most likely need to be developed by someone with the equipment to test it with.

**Comment by kermieisinthehouse** [comment]
From the survey: six people explicitly asked for buttplug.io support
